src/main/java/com/notably/logic/commands/ExitCommand.java

```diff
@@ -12,9 +15,12 @@
 public class ExitCommand extends Command {
     public static final String COMMAND_WORD = "exit";
 
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `ExitCommand`!